templates/telemetry/heartbeat/heartbeat.4.schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "heartbeat",
+  "type": "object",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
+    "payload": {
+      "additionalProperties": false,
+      "properties": {
+        "version": {
+          "maximum": 1,
+          "minimum": 1,
+          "type": "number"
+        },
+        "flowId": {
+          "type": "string"
+        },
+        "offeredTS": {
+          "type": "integer"
+        },
+        "learnMoreTS": {
+          "type": "integer"
+        },
+        "votedTS": {
+          "type": "integer"
+        },
+        "engagedTS": {
+          "type": "integer"
+        },
+        "closedTS": {
+          "type": "integer"
+        },
+        "expiredTS": {
+          "type": "integer"
+        },
+        "windowClosedTS": {
+          "type": "integer"
+        },
+        "score": {
+          "type": "integer"
+        },
+        "surveyId": {
+          "type": "string"
+        },
+        "surveyVersion": {
```

Comment:
Always a 64 chars hex

Reply:
my bad, i was going off of http://gecko-docs.mozilla.org.s3.amazonaws.com/toolkit/components/telemetry/telemetry/data/heartbeat-ping.html but apparently this stopped being an integer over a year ago. fixed.